src/main/java/org/opensearch/search/relevance/actionfilter/SearchActionFilter.java

```diff
@@ -230,52 +239,22 @@ public void onResponse(final Response response) {
-                    final SearchProfileShardResults profileResults = in.readOptionalWriteable(
-                            SearchProfileShardResults::new);
-                    final int numReducePhases = in.readVInt();
-
                     final SearchResponseSections internalResponse = new InternalSearchResponse(hits,
-                            aggregations, suggest,
-                            profileResults, timedOut, terminatedEarly, numReducePhases);
-
-                    final int totalShards = in.readVInt();
-                    final int successfulShards = in.readVInt();
-                    final int shardSearchFailureSize = in.readVInt();
-                    final ShardSearchFailure[] shardFailures;
-                    if (shardSearchFailureSize == 0) {
-                        shardFailures = ShardSearchFailure.EMPTY_ARRAY;
-                    } else {
-                        shardFailures = new ShardSearchFailure[shardSearchFailureSize];
-                        for (int i = 0; i < shardFailures.length; i++) {
-                            shardFailures[i] = readShardSearchFailure(in);
-                        }
-                    }
-
-                    final SearchResponse.Clusters clusters = new SearchResponse.Clusters(in.readVInt(),
-                            in.readVInt(), in.readVInt());
-                    final String scrollId = in.readOptionalString();
-                    final int skippedShards = in.readVInt();
-
-                    final long tookInMillis = (System.nanoTime() - startTime) / 1000000;
-                    final SearchResponse newResponse = new SearchResponse(internalResponse, scrollId,
-                            totalShards, successfulShards,
-                            skippedShards, tookInMillis, shardFailures, clusters);
+                            (InternalAggregations) searchResponse.getAggregations(), searchResponse.getSuggest(),
+                            new SearchProfileShardResults(searchResponse.getProfileResults()), searchResponse.isTimedOut(),
+                            searchResponse.isTerminatedEarly(), searchResponse.getNumReducePhases());
+
+                    final long tookInMillis = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime);
+                    final SearchResponse newResponse = new SearchResponse(internalResponse, searchResponse.getScrollId(),
+                            searchResponse.getTotalShards(), searchResponse.getSuccessfulShards(),
+                            searchResponse.getSkippedShards(), tookInMillis, searchResponse.getShardFailures(),
+                            searchResponse.getClusters());
                     listener.onResponse((Response) newResponse);
-
-                    // TODO: Change this to a metric
-                    logger.info("Result transformer operations overhead time: {}ms",
-                            tookInMillis - searchResponse.getTook().getMillis());
```

Comment:
Is there value in logging the total time spent on transforms? 

We output the time for individual transformers above. That doesn't cover the time spent e.g. fetching index settings.

Reply:
I think if we can log so we're separating time spent in a transformer vs time spent fetching index settings and doing other internal opensearch operations, that would be good because then a transformer that goes haywire could be investigated and tracked better. If you still have doubts, can you describe more?

Reply:
With this change, we are now logging the time that each individual transformer spent in the `transform` method, which would let us track down a single misbehaving transformer.

With the removal of the highlighted lines, we're no longer logging the time spent on everything else (e.g. loading index settings, the `shouldTransform` calls, the `preprocessRequest` calls).